### DISCUSSION TITLE
Post Console Error due to Magic Bell

### DISCUSSION THREAD

**Discussion by gerona21** [magicbell/community#181] [Questions] [answered]
Can someone can help me with this?

![Screenshot 2024-05-06 164816](https://github.com/magicbell-io/community/assets/162661984/d94cec13-4f7d-41b6-8801-3e25e41cc0ae)

**Comment by unamashana** [comment, marked as answer]
Please see https://github.com/orgs/magicbell-io/discussions/182#discussioncomment-9382192
